### ISSUE TITLE
Not triggered on different env?

### ISSUE THREAD

**Issue by demiro** [SolarLiner/vue-cli-plugin-prerender-spa#92]
**Describe the bug**
If I run the build with
npm run build -- --mode=prod (or any other env)

the prerender does not trigger.
How do I trigger it with different env settings?

**Comment by demiro** [comment]
YES, I was wondering the same.... please advise... why does not run on different mode?

**Comment by gogibogi4** [comment]
I have experienced the same issue. Running **npm run build -- --mode production** creates multiple html files while using **npm run build -- --mode development** creates only one html file. I have the onlyProduction option set to false in my vue.config.js. Has anyone found a solution to that?

**Comment by gogibogi4** [comment]
I have done some additional research on this. The problem is in **index.js** on **line 139**. The reason for this is linked with issue #16 . The issue itself is fixed with a workaround and has not been addresed directly. This should be properly fixed before reenabling the feature.

**Comment by ricky11** [comment]
I find  onlyProduction: false, does not work in dev mode,  (no compiled index.html is created) but does in production.
